cookbook/bundles/best_practices.rst

```diff
@@ -155,13 +142,13 @@ Classes
 
 The bundle directory structure is used as the namespace hierarchy. For
 instance, a ``HelloController`` controller is stored in
```

Comment:
This phrase looks confusing to me:

```
A HelloController is stored in HelloBundle/Controller/HelloController.php
and the fully qualified class name is ...
```

It seems like it's mandatory to store the controller in that exact path (but only the controller name is mandatory). What about changing it slightly by:

```
If a HelloController is stored in HelloBundle/Controller/HelloController.php,
the fully qualified class name is ...
```